cmd/atlasci/internal/ci/ci.go

```diff
@@ -182,3 +191,69 @@ func treeObject(r *git.Repository, n plumbing.ReferenceName) (*object.Tree, erro
 	}
 	return t, nil
 }
+
+// DevLoader implements the ChangesLoader interface using a dev-driver.
+type DevLoader struct {
+	// Dev environment used as a sandbox instantiated to the starting point (e.g. base branch).
+	Dev *sqlclient.Client
+	// Scan is used for scanning the migration directory.
+	Scan migrate.Scanner
```

Comment:
Why exported fields?

Reply:
Avoid creating a constructor at this stage. I may add one at some point.